README.md

```diff
@@ -1,9 +1,11 @@
 [![Build Status][build-badge]][build-status]
 [![Coverage Status][coverage-badge]][coverage-status]
 
-# Get It For Me (GIFM) Button Service
+# Get It For Me Button Service
 
-A service for interfacing with Library Catalogs developed and maintained by [Texas A&M University Libraries][tamu-library].
+A *Spring* backend for the *Get It For Me (GIFM) Button Service* developed and maintained by [Texas A&M University Libraries][tamu-library].
+
+This is a service for interfacing with *Library Catalogs*.
```

Comment:
Rather, "This is a service for providing links to the Illiad (https://www.atlas-sys.com/illiad/) request management system.  The links are customized based on the individual item in the Library catalog."

I know this is a first pass, but wanted to get this rather misleading statement out of there.